core/state_processor_rip7560.go

```diff
@@ -211,8 +176,19 @@ func ApplyRip7560ValidationPhases(chainConfig *params.ChainConfig, bc ChainConte
 		GasPrice: gasPrice,
 	}
 	evm := vm.NewEVM(blockContext, txContext, statedb, chainConfig, cfg)
+	epc := &EntryPointCall{}
+
+	if evm.Config.Tracer == nil {
+		evm.Config.Tracer = &tracing.Hooks{
+			OnEnter: epc.OnEnter,
+		}
+	} else {
+		// keep the original tracer's OnEnter hook
```

Comment:
add comment: Note that we modify an input parameter. We assume the Config.Tracer is an ephemeral parameter, created just for this call.